draft-tigress-requirements.md

```diff
@@ -57,6 +57,12 @@ author:
     name: Nick Sha
     organization: Alphabet Inc
     email: nicksha@google.com
+ -
+    ins: M. Gerster
+    name: Manuel Gerster
+    organization: Mercedes Benz
```

Comment:
@dimmyvi: Thx for adding me :) Just one request: Could you please change the organization to "Mercedes-Benz AG"?